experiences/branded-content/branded-content-card.test.js

```diff
@@ -16,27 +17,19 @@ const brandedContentTest = function() {
 	const fixtureUrl = getUrlFixture({}, 'https://playground.mrf.io/simulate?siteUrl=https://playground.marfeel.com/templates/branded-content-article.html&requestHostname=playground.marfeel.com&experienceUrl=croupier');
 
 	it('setup', async function() {
-		config = {
-			cards: {
-				BrandedContentFlowcard: {
-					content: {
-						type: 'AMPDocument',
-						// eslint-disable-next-line max-len
-						url: '${PLAYGROUND_PROXY}/${CURRENT_HOSTNAME}/experiences/branded-content/branded-content-article.html'
-					},
-					snapPoints: {
-						initial: 0.75,
-						active: 0.33
-					}
-				}
-			}
-		};
+		// eslint-disable-next-line max-len
+		config = await getFlowcardJson({}, 'https://flowcards.mrf.io/json/web?site_id=120&client_id=fec803ab-7426-4617-ab35-806db795b85c&canonical_url=https%3A%2F%2Fplayground.mrf.io%2Fsimulate%3FsiteUrl%3Dhttps%253A%252F%252Fplayground.marfeel.com%252Ftemplates%252Fbranded-content-article.html%26requestHostname%3Dplayground.marfeel.com%26experienceUrl%3Dcroupier&referrer=https%3A%2F%2Fplayground.marfeel.com&geo=__INJECT_GEO__&dynamic_flowcards=%5B%7B%22name%22%3A%22BrandedContentFlowcard%22%2C%22relevance%22%3A100%2C%22url%22%3A%22https%3A%2F%2Fplayground.marfeel.com%2Fexperiences%2Fbranded-content%2Fbranded-content-card.html%22%2C%22renderProps%22%3A%22%7B%5C%22content%5C%22%3A%7B%5C%22type%5C%22%3A%5C%22AMPDocument%5C%22%2C%5C%22url%5C%22%3A%5C%22https%3A%2F%2Fplayground.marfeel.com%2Fexperiences%2Fbranded-content%2Fbranded-content-card.html%5C%22%7D%2C%5C%22snapPoints%5C%22%3A%7B%5C%22initial%5C%22%3A0.75%2C%5C%22promoted%5C%22%3A176%2C%5C%22active%5C%22%3A210%7D%2C%5C%22features%5C%22%3A%7B%5C%22mode%5C%22%3A%5C%22modal%5C%22%2C%5C%22isDraggable%5C%22%3Atrue%2C%5C%22button%5C%22%3A%7B%5C%22action%5C%22%3A%5C%22setSnapPoint%3Ainitial%5C%22%7D%2C%5C%22removable%5C%22%3Atrue%2C%5C%22transition%5C%22%3A%5C%22scrollBounded%5C%22%2C%5C%22infiniteScroll%5C%22%3Atrue%2C%5C%22heroElement%5C%22%3A%7B%5C%22src%5C%22%3A%5C%22https%3A%2F%2Fplayground.marfeel.com%2Fassets%2Fexperiences%2Fheader-branded-dfp.png%3Fd1%5C%22%7D%7D%2C%5C%22triggers%5C%22%3A%7B%5C%22scrollTrigger%5C%22%3A%7B%5C%22on%5C%22%3A%5C%22scroll%5C%22%2C%5C%22spec%5C%22%3A%7B%5C%22type%5C%22%3A%5C%22absolute%5C%22%2C%5C%22pixel%5C%22%3A300%2C%5C%22percentage%5C%22%3A0.1%7D%7D%7D%7D%22%7D%5D');
+
 		fixture = {
 			url: fixtureUrl,
 			articleTitle: 'Article example'
 		};
 
 		await bootstrapExperience(browser, config, fixture);
+
+		//custom content url:
+		// eslint-disable-next-line max-len
+		config.cards.BrandedContentFlowcard.content.url = 'https://playground.marfeel/experiences/branded-content/branded-content-article.html';
```

Comment:
Should we generate our own templates to avoid production breaking changes make the tests fail?